test/integration/bank_holidays_test.rb

```diff
@@ -8,18 +8,18 @@ class BankHolidaysTest < ActionDispatch::IntegrationTest
 
     visit "/bank-holidays"
 
-    within 'head' do
-      assert page.has_selector?("title", :text => "UK bank holidays - GOV.UK")
-      desc = page.find("meta[name=description]")
+    within("head", visible: false) do
```

Comment:
@h-lame Unfortunately I couldn't find anything to make this less verbose. I came across an [article](http://makandracards.com/makandra/7617-change-how-capybara-sees-or-ignores-hidden-elements) that mentions using the global control `Capybara.ignore_hidden_elements = true`, but is generally advised against. It's also been suggested flipping this variable on and off before and after specific code that needs it, but I don't think that would be preferable to what we currently have.